### DISCUSSION TITLE
py-repl run button to be always visible

### DISCUSSION THREAD

**Discussion by limitcracker** [pyscript/pyscript#1401] [Proposals]
Is there a workaround in order to make the py-repl Run Button to be always visbile?

**Comment by JeffersGlass** [comment]
The button itself has the class `py-repl-run-button`, whose opacity is set to 0 by the default `pyscript.css`. You can override this rule by adding additional css styles in a later \<style\> tag:
```html
<style>
    .py-repl-run-button {
        opacity: 1 
    }
</style>
```
If this style tag comes after `<link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css">` in your page, it will take precedence over the default stylesheet. If you're using a templating system or otherwise where you're not confident of the tag ordering, you may want to use `opacity: 1 !important` to ensure it is higher precedence.
